src/Landing/reduxx/actions/LatestArticles.jsx

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
+import { FETCH_LATEST_POSTS } from '../types';
+
+const fetchLatestArticles = () => (dispatch) => {
+  axios.get('https://ah-backend-realers-staging.herokuapp.com/api/articles/?format=json&limit=9')
+    .then(res => res.data.results)
+    .then(articles => dispatch({
```

Comment:
I think you can use one `then` block with axios, unlike the fetch API...

Reply:
I have refactored this code to use one `then`